units/systemd-repart.service.in

```diff
@@ -16,17 +16,14 @@ ConditionDirectoryNotEmpty=|/usr/lib/repart.d
 ConditionDirectoryNotEmpty=|/usr/local/lib/repart.d
 ConditionDirectoryNotEmpty=|/etc/repart.d
 ConditionDirectoryNotEmpty=|/run/repart.d
-ConditionDirectoryNotEmpty=|/sysroot/usr/lib/repart.d
-ConditionDirectoryNotEmpty=|/sysroot/usr/local/lib/repart.d
-ConditionDirectoryNotEmpty=|/sysroot/etc/repart.d
 ConditionDirectoryNotEmpty=|/sysusr/usr/lib/repart.d
 ConditionDirectoryNotEmpty=|/sysusr/usr/local/lib/repart.d
 
 DefaultDependencies=no
 After=initrd-usr-fs.target
 Before=initrd-root-fs.target
-Conflicts=shutdown.target
-Before=shutdown.target
+Conflicts=shutdown.target initrd-switch-root.target
+Before=shutdown.target initrd-switch-root.target
```

Comment:
this change should be unnecessary, since we already have `Before=initrd-root-fs.target`, which is ordered before `initrd-switch-root.target` already, and pulled in by it.

Hence because of transitivity of ordering this change seems unnecessary.

Reply:
Yeah, it's unnecessary. But I think it's easier to keep it in, so that when looking at various units, you can immediately say that the ordering wrt. `initrd-switch-root.target` is there, without going through other units. (Also, there is no Wants/Requires on `initrd-root-fs.target`, so it's not strictly guaranteed that it'll be there.)